base/workflow/inst/batch_run.R

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env Rscript
-
-library(dplyr)
+library(tidyverse)
```

Comment:
The reason for the note is that you can't *import* from tidyverse into a package namespace (because it doesn't export anything!), not that there's anything wrong with *loading* it in an `inst` script. That said, there's a separate case to be made that if this script only uses a few of the Tidyverse packages then it's good to leave the others unloaded for namespace hygiene purposes. And if @ashiklom makes that case I won't disagree with it ;)